librz/include/sflib/freebsd-x86-32/sflib.h

```diff
@@ -0,0 +1,290 @@
+/*
```

Comment:
@XVilka should this also have the spdx header on top of the generated one?

Reply:
In theory - yes. But lets consider subproject-ing it first.